pkgs/servers/nosql/mongodb/mongodb.nix

```diff
@@ -121,7 +174,10 @@ in stdenv.mkDerivation rec {
     runHook postInstallCheck
   '';
 
-  installTargets = if (versionAtLeast version "4.4") then "install-core" else "install";
+  installTargets =
+    if (versionAtLeast version "6.0") then "install-devcore"
```

Comment:
Hmm, may be interesting if we standardize around a single installation target in the future.

Reply:
We can't they changed default name during version it seems. We can have a slimmer mongo v5 with different target but don't know if people were relying on it so didn't changed the name.

Reply:
The `mongo` executable used in the tests (and useful on its own) moved to
`install-devcore` in 6.0.

Those are moving targets :)